client/package.json

```diff
@@ -3,12 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "grommet": "^2.5.5",
-    "react": "^16.8.3",
-    "react-dom": "^16.8.3",
-    "react-scripts": "2.1.5",
-    "socket.io-client": "^2.2.0",
-    "styled-components": "^4.1.3"
+    "grommet": "^2.21.0",
```

Comment:
Hey, could you please remove all of the `^`s here and then redo an install and commit the package lock changes? The `^` will make npm automatically update minor + patch versions. It might be worth also have a look on the npm site to see if there's any patch versions we can upgrade here too.

Reply:
Thanks JT. I've always thought the `^` was a "pin". Turns out it's the exact opposite.

Updated those and bumped a few more packages to their latest versions on NPM.
I'm keeping `react` at 17.0.2, it looks like `grommet` expects up to 17 only

```
grommet@2.22.0 requires a peer of react@^16.6.1 || ^17.0.0 but none is installed. You must install peer dependencies yourself.
```